community.md

```diff
@@ -170,6 +170,47 @@ Make sure to tag your questions there accordingly to get answers from the Flink
 ## Issue Tracker
 
 We use Jira to track all code related issues: [{{ site.jira }}]({{ site.jira }}).
+You must have a JIRA account in order to log cases and issues.
+
+### I already have an ASF JIRA account and want to be added as a contributor
+
+If you already have an ASF JIRA account, you do not need to sign up for a new account. 
+Please email [jira-requests@flink.apache.org]([jira-requests@flink.apache.org]) using the following template, so that we can add your account to the 
+contributors list in JIRA:
+
+[Open the template in your email client](mailto:jira-requests@flink.apache.org?subject=Add%20me%20as%20a%20contributor%20to%20JIRA&amp;body=Hello,%0A%0APlease%20add%20me%20as%20a%20contributor%20to%20JIRA.%0AMy%20JIRA%20username%20is:%20%5BINSERT%20YOUR%20JIRA%20USERNAME%20HERE%5D%0A%0AThanks,%0A%5BINSERT%20YOUR%20NAME%20HERE%5D)
+
+```text
+Subject: Add me as a contributor to JIRA
+
+Hello,
+
+Please add me as a contributor to JIRA.
+My JIRA username is: [INSERT YOUR JIRA USERNAME HERE]
+
+Thanks,
+[INSERT YOUR NAME HERE]
+```
+
+### I do not have an ASF JIRA account, want to request an account and be added as a contributor
+
+In order to request an ASF JIRA account, you will need to email [jira-requests@flink.apache.org]([jira-requests@flink.apache.org]) using the following template:
+
+[Open the template in your email client](mailto:jira-requests@flink.apache.org?subject=Request%20for%20JIRA%20Account&amp;body=Hello,%0A%0AI%20would%20like%20to%20request%20a%20JIRA%20account.%0AMy%20proposed%20JIRA%20username:%20%5BINSERT%20YOUR%20DESIRED%20JIRA%20USERNAME%20HERE%20(LOWERCASE%20LETTERS%20AND%20NUMBERS%20ONLY)%5D%0AMy%20full%20name:%20%5BINSERT%20YOUR%20FULL%20NAME%20HERE%5D%0AMy%20email%20address:%20%5BINSERT%20YOUR%20EMAIL%20ADDRESS%20HERE%5D%0A%0AThanks,%0A%5BINSERT%20YOUR%20NAME%20HERE%5D)
+
+```text
+Subject: Request for JIRA Account
+
+Hello,
+
+I would like to request a JIRA account.
+My proposed JIRA username: [INSERT YOUR DESIRED JIRA USERNAME HERE (LOWERCASE LETTERS AND NUMBERS ONLY)]
```

Comment:
User need try this process again if username conflicts happens? 
Could we offer guidance to check the desired username will be conflicted with existed usernames or not?

Reply:
Let's see if we run into a lot of conflicts first. If we do, then let's change the docs with a little howto check if the username is already taken

Reply:
Okay, it makes sense to me